app/controllers/concerns/idv_step_concern.rb

```diff
@@ -108,6 +108,8 @@ def flow_policy
   end
```

Comment:
I wonder if I'm missing a setup step, but when I check out this branch and try running through the testing steps, I did not see the expected result.

Specifically, on desktop, I was not re-directed to the hybrid handoff page. Instead I was able to continue to the upload page.

Reply:
@night-jellyfish , you have to set `doc_auth_selfie_desktop_test_mode: false`